### PR TITLE
Add participant name to Participant Canvas

### PR DESCRIPTION
**Pull request by mythilytm**
Primary reviewer:

## Description
- Adds participant name to Participant Canvas

### Checklist
- [x] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [x] Tested for call contacts

### Related Issues
Fixes # https://tech-matters.atlassian.net/browse/CHI-1938?focusedCommentId=12913

### Verification steps
Screenshot 2023-06-22 at 11 52 04 AM

### CODE REVIEW

File: plugin-hrm-form/src/components/Conference/ConferenceActions/PhoneInputDialog.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@
  * along with this program.  If not, see https://www.gnu.org/licenses/.
  */
 import React from 'react';
+import { useSelector } from 'react-redux';
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/Conference/ConferenceActions/PhoneInputDialog.tsx:17]: Is this used?

File: plugin-hrm-form/src/components/Conference/HoldParticipantButton/index.tsx [hunk 2]
```diff
@@ -15,15 +15,19 @@
...
 import type { ParticipantCanvasChildrenProps } from '@twilio/flex-ui/src/components/canvas/ParticipantCanvas/ParticipantCanvas.definitions';
 
+import { conferencingBase, namespace, RootState } from '../../../states';
 import { conferenceApi } from '../../../services/ServerlessService';
 
 type Props = Partial<ParticipantCanvasChildrenProps>;
 
-const HoldParticipantButton: React.FC<Props> = ({ participant, task }) => {
+const HoldParticipantButton: React.FC<Props> = ({ participant, task, ...props }) => {
+  console.log('>>> HoldParticipantButton', participant, task, props);
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/Conference/HoldParticipantButton/index.tsx:28]: Let's remove this before merging.

File: plugin-hrm-form/src/components/Conference/HoldParticipantButton/index.tsx [hunk 3]
```diff
@@ -40,6 +44,17 @@ const HoldParticipantButton: React.FC<Props> = ({ participant, task }) => {
     setIsLoading(false);
   };
 
+  // Adds phone number to participants in ParticipantCanvas
+  ParticipantCanvas.Actions.Content.add(
+    <p key="participant-name">
+      <strong>{phoneNumber}</strong>
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/Conference/HoldParticipantButton/index.tsx:50]: I don't think you want to show this here, but the participant call address (which is a property of the participant itself). This is determined by an input and will change, specially when in 4 ways conferences.

**Review by GPaoloni — COMMENTED** [review]
Some comments on this one, should be minor but need to be addressed.

**Comment by mythilytm** [comment]
Closing because there is a better solution

**Pull request closed without merging**